packages/ui/src/components/controls/Controls.module.scss

```diff
@@ -286,3 +286,22 @@
     }
   }
 }
+
+.infoBox {
+  background-color: var(--input-background);
+  color: rgba(255, 255, 255, 0.6);
+  border: 0;
+  border-radius: var(--radius);
+  padding: 0 8px;
+  height: 24px;
+  flex-shrink: 1;
+  flex-grow: 1;
+  min-width: 0;
+  font-weight: bold;
+  display: block;
+  font-family: var(--font-family-mono);
+}
```

Comment:
What's the difference between this and the `<Input>` component used for displaying framerate for example?

Reply:
I could not figure out how to resize the input component so it was only the length of the text. That is the only difference.

Reply:
I'd still use the `.input` class then, just with a `div` element.
The size being constant is caused by the `input` element itself and not the styling so it should be possible to use the same class.

Reply:
Fair point. I just styled a div with the same `.input` class and remove my InfoBox component.